message_ix/cli.py

```diff
@@ -39,9 +40,9 @@ def copy_model(path, overwrite, set_default):
     path = Path(path).resolve()
 
     src_dir = Path(__file__).parent / "model"
-    for src in src_dir.rglob("*"):
+    for src in src_dir.rglob("*.*"):
         # Skip certain files
-        if src.suffix in (".gdx", ".log", ".lst"):
+        if src.suffix in (".gdx", ".log", ".lst") or re.search("(225)", src.stem):
```

Comment:
I suspect this won't work, since the "stem" is only the part of the filename before the suffix. E.g. in /path/to/files/225a/etc/foo.gdx, the stem is "foo". We want to check if any entire part is `225[a-z]{1,2}`.

Can you experiment locally? Later (not today) I can show how to extend the tests to check this.

Reply:
Thank you for the review!
Ugh, yes, I checked now - it's not working. Apparently I checked something else before pushing which worked.. Sorry for that.

> Can you experiment locally?

Yes!

> Later (not today) I can show how to extend the tests to check this.

Thank you! Will also look into that before.

Reply:
Checking locally with the current adjustment, a folder called "225" gets skipped, but maybe there is a better way do do that. 
I've tried to implemented a test to check if this behavior is working, I guess this could also be optimized.